diff -Naupr auks-0.5.0.old/etc/auksdrenewer.service.in auks-0.5.0/etc/auksdrenewer.service.in
--- auks-0.5.0.old/etc/auksdrenewer.service.in	2020-06-16 20:04:00.000000000 -0400
+++ auks-0.5.0/etc/auksdrenewer.service.in	2020-09-21 13:32:55.033494435 -0400
@@ -5,7 +5,7 @@ Wants=aukspriv.service
 
 [Service]
 Type=simple
-EnvironmentFile=-/etc/sysconfig/auksdrenewer
+EnvironmentFile=-/etc/conf.d/auksdrenewer
 ExecStart=@sbindir@/auksdrenewer -F $AUKSDRENEWER_OPTIONS
 ExecReload=/bin/kill -HUP $MAINPID
 Restart=on-failure
diff -Naupr auks-0.5.0.old/etc/auksd.service.in auks-0.5.0/etc/auksd.service.in
--- auks-0.5.0.old/etc/auksd.service.in	2020-06-16 20:04:00.000000000 -0400
+++ auks-0.5.0/etc/auksd.service.in	2020-09-21 13:33:31.723548247 -0400
@@ -4,7 +4,7 @@ After=network.target
 
 [Service]
 Type=simple
-EnvironmentFile=-/etc/sysconfig/auksd
+EnvironmentFile=-/etc/conf.d/auksd
 ExecStart=@sbindir@/auksd -F $AUKSPRIV_OPTIONS
 ExecReload=/bin/kill -HUP $MAINPID
 Restart=on-failure
diff -Naupr auks-0.5.0.old/etc/aukspriv.service.in auks-0.5.0/etc/aukspriv.service.in
--- auks-0.5.0.old/etc/aukspriv.service.in	2020-06-16 20:04:00.000000000 -0400
+++ auks-0.5.0/etc/aukspriv.service.in	2020-09-21 13:33:52.283578403 -0400
@@ -4,7 +4,7 @@ After=network.target
 
 [Service]
 Type=forking
-EnvironmentFile=-/etc/sysconfig/aukspriv
+EnvironmentFile=-/etc/conf.d/aukspriv
 ExecStart=@sbindir@/aukspriv $AUKSPRIV_OPTIONS
 GuessMainPID=true
 
